discovery-infra/test_infra/helper_classes/config/controller_config.py

```diff
@@ -0,0 +1,33 @@
+from abc import ABC
+from pathlib import Path
+from dataclasses import dataclass
+
+from test_infra import consts
+from .base_config import _BaseConfig
+
+
+@dataclass
+class ControllerConfig(_BaseConfig, ABC):
```

Comment:
Rename to `BaseControllerConfig` to keep with the conventions

Reply:
The name should be nodes config, no? This is not just a generic config that would be used from any future controller.

Reply:
It's a common controller configurations not a node configuration